packages/statistics-module/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@platzily/statistics-module",
+  "version": "1.0.0",
+  "description": "Statistics Services",
+  "author": "Statistics Team <@platzily/statistics>",
```

Comment:
We could add also contributors, template in https://github.com/platzily/platzily-monolith/pull/39#discussion_r697885170